backend/account_v2/tests/test_last_admin_guard.py

```
@@ -0,0 +1,228 @@
+        INSTALLED_APPS=[],
+    )
+
+
+# ---------------------------------------------------------------------------
+# Hand-rolled service stub — avoids any mock framework dependency.
+# ---------------------------------------------------------------------------
+
+
+class _OrgMemberServiceStub:
+    _member_by_email: SimpleNamespace | None = None
+    _all_members: list = []
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)